### PR TITLE
Add live-logs plugin to JFrog CLI plugins registry

### PR DESCRIPTION
**Pull request by amithins**

### CODE REVIEW

File: plugins/live-logs.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# The live-logs plugin
+pluginName: live-logs
+version: v1.0.0
+repository: https://github.com/jfrog/live-logs
+maintainers:
+  - amithins
+  - eyalbe4
```
Comment (eyalbe4) [inline plugins/live-logs.yml:7]: Please remove *eyalbe4*, since he is not the maintainer of the plugin.

File: plugins/live-logs.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+tag: v1.0.0
```
Comment (eyalbe4) [inline plugins/live-logs.yml:8]: It looks like tag v1.0.0 does not exist on ```https://github.com/jfrog/live-logs```.
This will cause the automatic fail.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
See the comment added by ```CLA Assistant Lite bot``` .
Signing the terms will trigger the automatic plugin of validations.

**Comment by eyalbe4** [comment]
@amithins,
It looks like the registry experienced a temporary issue, causing the validations not to be triggered following your commit.
Will you be able to push another commit to this PR, adding a new-line into ```live-logs.yml```?
This should trigger the validations.

**Comment by amithins** [comment]
I have read the Developer Terms and I hereby accept the terms

**Comment by amithins** [comment]
recheck

**Comment by eyalbe4** [comment]
@amithins,

live-logs v1.0.0 has been published and can be installed using JFrog CLI. 
Impressive plugin! Great work!!

I suggest improving the plugin's description. See below - the description is currently **live-logs - A new cli application**.
You can fix it and register a new patch to the registry.
We'll soon also update the registry's README to include the new plugin.

```
$ jfrog live-logs --help
NAME:
   live-logs - A new cli application

USAGE:
   live-logs [global options] command [command options] [arguments...]
   
VERSION:
   v1.0.0
   
COMMANDS:
   logs, l    
   config, c  
   help, h    Shows a list of commands or help for one command
   
GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version
   

```

**Pull request merged**